kong-0.2.0-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong"
-version = "0.1.1beta-2"
+version = "0.2.0-1"
 supported_platforms = {"linux", "macosx"}
 source = {
   url = "git://github.com/Mashape/kong",
-  tag = "0.1.1beta-2"
```

Comment:
But the thing is that, as already discussed, Luarocks doesn't support semver. We have to either do `1.0-1` or `1.0.0-1` (I don't even think it support `-0` iirc). It is not really npm. That is something we could maybe discuss with Luarocks too.